packages/client/src/views/CorrectionForm/VerifyCorrector.tsx

```diff
@@ -78,7 +79,7 @@ class VerifyCorrectorComponent extends React.Component<IFullProps> {
       ...application,
       data: {
         ...application.data,
-        corrector: {
+        correction: {
```

Comment:
Could you please explain why do we need this?

Reply:
Sorry for the confusion @Zangetsu101 , this change may be unintentional

Reply:
Removed at [6221bbb](https://github.com/opencrvs/opencrvs-core/pull/2585/commits/6221bbb8778dfaa66df3e601390e48f05dcf92e7)